### DISCUSSION TITLE
Secret signature different across examples

### DISCUSSION THREAD

**Discussion by gyroscopes** [dojo-engineering/feedback#5] [Docs]
Minor thing but the secret signature across the examples is different. And also different from the written example in the documentation. 
Would be nice if it matched up, mainly for testing the webhooks.

**Comment by RomanKondratyev** [comment]
This is minor indeed, because it is always random, but we could align to decrease confusion while reading the docs. @monica-m-ps

**Comment by RomanKondratyev** [comment]
Adjusted, feel free to point out other places.
